python-plots/plot.py

```diff
@@ -0,0 +1,71 @@
+import matplotlib
+import matplotlib.pyplot as plt
+import scipy.integrate as integrate
+from scipy.interpolate import interp1d
+import csv
+import math
+
+# Config for the plot
+font = {'family' : 'Roboto',
+        'weight' : 'normal',
+        'size'   : 20}
+matplotlib.rc('font', **font)
+
+# Constants
+PEAK_START_TIME = 150 # !HARDCODED! Time when peak starts (manually derived, [s])
+PEAK_END_TIME = 300 # !HARDCODED! Time when peak ends (manually derived, [s])
+
+# Variables
+baseline = math.inf
+
+x_vals = []
+y_vals = []
+
+peak_x_vals = []
+peak_y_vals = []
+
+# Read data !HARDCODED! Path to CSV data
+with open('raw-data-a.csv', newline='') as csvfile: # File must be formatted as: <time as float>,<signal as float>\n
+    reader = csv.reader(csvfile, delimiter=',')
+    # Iterate over each line and populate list
+    for row in reader:
+        # Read data and parse to float
+        TIME = float(row[0]) * 60 # Time val is given as minute (float), hence convert to seconds
+        SIGNAL = float(row[1])
+
+        x_vals.append(TIME)
+        y_vals.append(SIGNAL)
+
+# Derive peak values
+i = -1
+for y_val in y_vals:
+    i = i + 1
+
+    if x_vals[i] < PEAK_START_TIME or x_vals[i] > PEAK_END_TIME: # Skip if time is not in between PEAK_START_TIME and PEAK_END_TIME
```

Comment:
Here I create a subset of the original data, that is, the peak itself limited by `PEAK_START_TIME` and `PEAK_END_TIME`. As you already said: This is subjective, but it is precise enough. The constants are defined at the beginning of the script